### PR TITLE
Channel labs

### PR DESCRIPTION
**Pull request by niwinz**
Thanks to @yurrriq for the initial port of the funcool/canal to use conditional readers.

### CODE REVIEW

**Comment by niwinz** [comment]
You are completely right! Now is fixed!

**Comment by yurrriq** [comment]
:tada:

File: doc/content.adoc [hunk 1]
```diff
@@ -1255,6 +1256,130 @@ the `curry-lift-m` macro. The arity is mandatory when using this macro:
...
+
+And here we can observe few things:
+
+* The asynchronous calls are made in parallel.
+* The total time of processing is half less of if we use `mlet`.
+* The `return` function is not used because `alet` evaluates the body in the context
+  of the applicative.
+
+The alet is a powerfull macro that analyzes the dependencies between bindings and executes
+the expressions in badges resultin in a very atractive feature for asynchronous calls.
```
Comment (dialelo) [inline doc/content.adoc]: s/badges/batches

**Comment by dialelo** [comment]
Looks good to me apart from the small typo I spotted!

**Comment by niwinz** [comment]
Thanks! I'll fix it and merge later.

**Comment by dialelo** [comment]
:+1:

**Pull request merged**